pkg/startup/startup.go

```diff
@@ -60,6 +61,7 @@ const (
 	AUTODETECTION_METHOD_CAN_REACH      = "can-reach="
 	AUTODETECTION_METHOD_INTERFACE      = "interface="
 	AUTODETECTION_METHOD_SKIP_INTERFACE = "skip-interface="
+	AUTODETECTION_METHOD_CIDR           = "cidr="
```

Comment:
I think this should be called `cidrs` since it's a list

Reply:
I guess our other options here are singular, even though they support multiple options. So you can ignore this comment.